test/schemas/test/playbooks/ignore-unreachable.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Test
+  hosts: localhost
+  tasks:
+    - name: Debug
+      ansible.builtin.debug:
+        msg: ignore_unreachable should be a boolean
+      ignore_unreachable: true
```

Comment:
I think that the bug was about using jinja with ignore_unreachable, poor booleans did  work. Please be sure you include some jinja instead.

Reply:
@ssbarnea it's still failing and I'm trying to fix the same

Reply:
I thought the PR was in draft!